docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{data-sources-cap} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data-source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, Timelines, or Hosts.
```

Comment:
A few tweaks here. I know the previous para also mentioned "event or alert data," but I think it's helpful to reiterate for clarity. To a new user, every page in the app probably looks like it contains some kind of data.
```suggestion
You can tell which {data-source} is active by clicking the *{data-source-cap}* menu at the upper right of {es-sec} pages that display event or alert data, such as Overview, Alerts, Timelines, or Hosts.
```